server/questions.js

```diff
@@ -59,7 +59,7 @@ var prepareQuestionData = function(question, callback){
     questionToAdd.mtime = currentDate;
     questionToAdd.ratings = [];
     questionToAdd.comments = [];
-
+    questionToAdd.userSubmissionTime = [];
     //Add specific attributes by Type
```

Comment:
Can we have it named lockedUsers plz. UserSubmissionTime does not reflect on the actual functionality of the locking feature. Also if we decide to change the locking into number of tries this name won't make any sense then

Reply:
I don't like the variable lockedUser since the information stored in it doesn't mean the user is locked... the information stored is the users last submission time.

Reply:
The purpose of the list if to hold information about locked users. It shouldn't say anything about how they are locked or why, just simply that they are locked. Idk why you think the name doesn't fit. But if you feel that this names fits it then sure go for it.